static/js/ducks/sources.js

```diff
@@ -58,6 +67,25 @@ export function fetchFavoriteSources(filterParams = {}) {
   return API.GET("/api/sources", FETCH_FAVORITE_SOURCES, filterParams);
 }
 
+export function fetchGcnEventSources(dateobs = null, filterParams = {}) {
+  addFilterParamDefaults(filterParams);
+  filterParams.localizationDateobs = dateobs;
+  filterParams.localizationCumprob = 0.95;
```

Comment:
Why does this default to 0.95 here but 1.01 on the backend?

Reply:
0.95 makes more sense indeed. Fixing.